host/activity_test.go

```diff
@@ -921,7 +922,7 @@ func (s *integrationSuite) TestActivityHeartbeatTimeouts() {
 
 	s.Logger.Info("StartWorkflowExecution", tag.WorkflowRunID(we.RunId))
 
-	workflowComplete := false
+	workflowComplete := &atomic.Bool{}
```

Comment:
Use atomic.

Reply:
I'm not sure how turning this into atomic helps.  This variable is never accessed concurrently.

Reply:
My understanding is `wtHandler` and `atHandler` works in parallel. Am I wrong?

Reply:
They are only invoked when we poll for tasks, which is done sequentially in the test.